qutebrowser/browser/commands.py

```diff
@@ -1175,6 +1175,25 @@ def tab_move(self, index: typing.Union[str, int] = None, count=None):
         cmdutils.check_overflow(new_idx, 'int')
         self._tabbed_browser.tabBar().moveTab(cur_idx, new_idx)
 
+    @cmdutils.register(instance='command-dispatcher', scope='window')
+    @cmdutils.argument('choice', completion=miscmodels.suggest)
+    def suggest(self, command: str, columns: int,
+                sep_suggestions: str, sep_columns: str,
+                suggestions: str, choice: str):
+        """Allows userscripts to let a user choose between extracted information.
+
+        Args:
+            command: Qutebrowser command to run with the choice
+            columns: self-explanatory
+            sep_suggestions: charsequence which separates suggestions
+            sep_columns: charsequence which separates columns
+            suggestions: self-explanatory
```

Comment:
This definitely wasn't self-explanatory to me at first glance, it took me a while to figure out what all these args mean (with the assistance of reading the code).